spire/journal/actions.py

```diff
@@ -462,13 +462,24 @@ async def create_journal_entry(
         journal_spec=entry_request.journal_spec,
         user_group_id_list=user_group_id_list,
     )
+
+    if entry_request.created_at:
+        entry_request.tags.append(f"reported_at:{datetime.utcnow()}")
+        if entry_request.created_at > datetime.utcnow():
```

Comment:
I don't think we should change `created_at` if user specified it. It is their decision what data to put there (as long as it is datetime).